### PR TITLE
Html fix

### PR DESCRIPTION
**Pull request by dhonx**

### CODE REVIEW

File: play.html [hunk 1]
```diff
@@ -331,21 +324,20 @@
...
+	BLUE 		= rgb(0, 0, 255)
 )
 
 fn main() {
-        println(NUMBERS)
-        println(RED)
-        println(BLUE)
+	println(NUMBERS)
+	println(RED)
+  println(BLUE)
```
Comment (vbrazo) [inline play.html]: Indentation issue

**Comment by medvednikov** [comment]
@donnisnoni95 merge conflict

File: play.html (hunk 1 above)
Comment (donnisnoni95) [reply play.html]: I think we can merge it now... because of its looks not like this on me...

**Comment by donnisnoni95** [comment]
@medvednikov Done !!  👌

**Pull request closed without merging**